recipes/nginx/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "nginx" %}
+{% set version = "1.11.12" %}
+{% set sha256 = "2aff7f9396d1f77256efc363e1cc05ba52d40a29e6de4d9bc08aa444eea14122" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://nginx.org/download/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - pcre-config.patch
+
+build:
+  number: 0
+  no_link:
+    - etc/*
+    - var/log/nginx/*
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - libgd
+    - libxslt
+    - openssl 1.0.2*
+    - pcre
+    - zlib
```

Comment:
Please check the pinnings used in conda-forge and apply them here.

See https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L28-L71